sdk/resources/Azure.ResourceManager.Resources/api/Azure.ResourceManager.Resources.netstandard2.0.cs

```diff
@@ -17,7 +17,7 @@ public partial class ArmApplicationCollection : Azure.ResourceManager.ArmCollect
         System.Collections.Generic.IEnumerator<Azure.ResourceManager.Resources.ArmApplicationResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.Resources.ArmApplicationResource>.GetEnumerator() { throw null; }
         System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
     }
-    public partial class ArmApplicationData : Azure.ResourceManager.Resources.Models.ArmApplicationResource
+    public partial class ArmApplicationData : Azure.ResourceManager.Resources.Models.ArmApplicationResourceInfo
```

Comment:
We should rename this to `ArmApplicationResourceData` according to @m-nash 's suggestion